steps.md

```
@@ -2,35 +2,59 @@
+| [SearchApiTrait](#searchapitrait) | Assert Drupal Search API with index and query operations. |
+| [TaxonomyTrait](#taxonomytrait) | Manage Drupal taxonomy terms with vocabulary organization. |
+| [TestmodeTrait](#testmodetrait) | Configure Drupal Testmode module for controlled testing scenarios. |
+| [UserTrait](#usertrait) | Manage Drupal users with role and permission assignments. |
+| [WaitTrait](#waittrait) | Waiting for a period of time or for AJAX to finish. |
+| [WatchdogTrait](#watchdogtrait) | Assert Drupal does not trigger PHP errors during scenarios using Watchdog. |
+
+## BigPipeTrait
+
+[Source](src/BigPipeTrait.php), [Example](tests/behat/features/big_pipe.feature)
+
+>  Bypass Drupal BigPipe when rendering pages.
```

markdownlint-cli2 (0.17.2)

Multiple spaces after blockquote symbol
null

(MD027, no-multiple-space-blockquote)